lib/Db/SignRequestMapper.php

```diff
@@ -416,6 +416,41 @@ public function getFilesAssociatedFilesWithMeFormatted(
 		return $return;
 	}
 
+	public function getFilesToSearchProvider(IUser $user, string $fileName, int $limit, int $offset): array {
+		$filter = [
+			'page' => ($offset / $limit) + 1,
+			'length' => $limit,
+			'fileName' => $fileName,
+		];
+
+		$sort = [
+			'sortBy' => 'created_at',
+			'sortDirection' => 'desc',
+		];
+
+		$qb = $this->getFilesAssociatedFilesWithMeQueryBuilder($user->getUID(), $filter, false, $sort);
+
+
+		$qb->orderBy('f.created_at', 'DESC');
```

Comment:
Look, now the $sort is sent at row 431  but here is defined again. This row should be removed now
```suggestion
```